frontend/src/components/BCHeaderwSide/index.tsx

```diff
@@ -60,7 +60,7 @@ const listItems: ListItems[] = [
           },
           {
             name: 'Silviculture search',
-            link: '/opening/silviculture',
+            link: '/search',
```

Comment:
question: maybe `/search` is to general? How about `/silviculture-search` ?

Reply:
done, changed to silviculture-search